Source/Basic Shapes/SvgPolygon.cs

```diff
@@ -39,7 +39,7 @@ public override GraphicsPath Path(ISvgRenderer renderer)
                         // i.e stroke width only to be considered when calculating boundary
                         if (renderer == null)
                         {
-                            var radius = base.StrokeWidth / 2;
+                            var radius = base.StrokeWidth * 2;

```

Comment:
can you please explain why the radius is the double stroke width? I'm sure it is correct as it works, but I don't understand it...

Reply:
Here is an example SVG:

```svg
<?xml version="1.0" encoding="utf-8"?>
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" width="400" height="400">

<polygon points="200,200 100,200 200,144.6" style="opacity:0.9;fill:red;stroke:black;stroke-width:40;stroke-opacity:0.8;fill-opacity:1.0;" />

<circle cx="100" cy="200" r="20" style="opacity:0.6;fill:blue;" />

<circle cx="100" cy="200" r="40" style="opacity:0.6;fill:blue;" />
<circle cx="100" cy="200" r="80" style="opacity:0.6;fill:blue;" />

<circle cx="200" cy="200" r="1" style="fill:white;" />
<circle cx="100" cy="200" r="1" style="fill:white;" />
<circle cx="200" cy="144.6" r="1" style="fill:white;" />

</svg>
```

Although the calculation including stroke is very complicated, it can not do everything...

Reply:
Ah, thanks - I didn't get that this was for the bounds calcualation only (reading comments sometimes helps...)